manifests/ca.pp

```diff
@@ -104,6 +107,7 @@
   $key_name     = '',
   $key_ou       = '',
   $tls_auth     = false,
+  $easyrsa_source = $::openvpn::params::easyrsa_source,
```

Comment:
We already handle [different versions of Ubuntu in `params.pp`](https://github.com/luxflux/puppet-openvpn/blob/master/manifests/params.pp#L68-L87) and it's especially always about this path (which seems to change for every new release...), so there's everything in place to add the path for v13.04, IMHO?
Or do I misunderstand the issue?